src/providers/authentication/eIDProvider.ts

```diff
@@ -0,0 +1,116 @@
+import {Dispatch, SetStateAction} from 'react';
+import {addMessageListener, AusweisAuthFlow, AusweisSdkMessage, sendCommand} from '@animo-id/expo-ausweis-sdk';
+import {PARMode} from '@sphereon/oid4vci-common';
+import {OpenID4VCIClient} from '@sphereon/oid4vci-client';
+import {EIDFlowState, EIDGetAuthorizationCodeArgs, EIDHandleErrorArgs, EIDInitializeArgs, EIDProviderArgs} from '../../types';
+
+class EIDProvider {
```

Comment:
Suggest to call this something like a PIDServiceGermany. It is all about PIDs

Reply:
changed to PIDServiceGermany